README.rst

```diff
@@ -8,6 +8,6 @@ To run::
 	$ sudo pip install uwsgi
 	$ virtualenv -p python3 /var/tmp/deckhand
 	$ . /var/tmp/deckhand/bin/activate
-	$ pip install .
+	$ sudo pip install .
```

Comment:
I guess this sudo slipped back in.  Maybe we can merge some of your older PRs once they're out of merge conflict so that you can rebase onto a more current master (I assume that's what happened)?